nuxeo-coldstorage-web/ftest/features/coldstorage.feature

```diff
@@ -0,0 +1,54 @@
+Feature: Cold Storage
+
+  Background:
+    Given user "Jack" exists in group "members"
+
+  Scenario: Give WriteColdStorage permission to user
+    Given I login as "Administrator"
+    And I have a File document
+    And This document has file "sample.png" for content
+    When I browse to the document
+    And I give writeColdStorage permission to "Jack" on the document
+    Then I can see that "Jack" has the WriteColdStorage permission
+    
+  Scenario: User with WriteColdStorage permission can send file to cold storage
+    Given I login as "Jack"
+    And I have a File document
+    And This document has file "sample.png" for content
+    And I have permission WriteColdStorage for this document
+    When I browse to the document
+    And I send the file to cold storage
+    Then I can see the file is stored in cold storage
+
+  Scenario: User with ReadWrite permission can't send file to cold storage
+    Given I login as "Jack"
+    And I have a File document
+    And This document has file "sample.png" for content
+    And I have permission ReadWrite for this document
+    When I browse to the document
+    Then I cant send the file to cold storage
+
+  Scenario: User can send multiple files to cold storage in Workspace view
+    Given I login as "Jack"
+    And I have the following documents
+      | doctype    | title    | path                | file        |
+      | Workspace  | ws       | /default-domain     |             |
+      | File       | sample1  | /default-domain/ws  | sample.png  |
+      | File       | sample2  | /default-domain/ws  | sample.png  |
+      | File       | sample3  | /default-domain/ws  | sample.png  |
+    And I have the following permissions to the documents
+      | permission        | path                        |
+      | WriteColdStorage  | /default-domain/ws/sample1  |
+      | WriteColdStorage  | /default-domain/ws/sample2  |
+      | WriteColdStorage  | /default-domain/ws/sample3  |
+    When I browse to the document with path "/default-domain/ws"
+    And I select the "sample1" document
+    And I select the "sample2" document
+    And I select the "sample3" document
+    And I move the files to cold storage
+    When I browse to the document with path "/default-domain/ws/sample1"
+    Then I can see the file is stored in cold storage
+    When I browse to the document with path "/default-domain/ws/sample2"
+    Then I can see the file is stored in cold storage
+    When I browse to the document with path "/default-domain/ws/sample3"
+    Then I can see the file is stored in cold storage
```

Comment:
I am wondering if we have to check we have the popup before to validate the move to cold storage.

Reply:
What pop-up? The confirmation dialog?

Reply:
yes,

Reply:
That is included inside the steps `I move the files to cold storage` and `I send the file to cold storage`.
```
  const dialog = this.ui.browser.el.element('nuxeo-dialog#contentsToMoveDialog');
  dialog.waitForVisible();
  dialog.click('paper-button[name="confirm"]');
```

Reply:
yes, I know I saw it. But is it possible to add a specific scenario to check its presence. Thanks

Reply:
There is no scenario in which the confirmation dialog appears without clicking on the action first, and there is no scenario in which I click on the action and I don't expect the dialog to appear. I don't understand why we need a specific step for the confirmation.

Reply:
What I had in mind is the check the popup dialog behaviour.
- Visible -> for example the scenrio will be: 

```
Scenario: User with WriteColdStorage permission must see popup dialogue confirmation before sending file to cold storage
    Given I login as "Jack"
    And I have a File document
    And This document has file "sample.png" for content
    And I have permission WriteColdStorage for this document
    When I browse to the document
    And I click the action button 
    Then I should see a pop dialogue for confirmation
```
thing like that, it's like what you did for Aspera. 



- Accept to send the content (exactly what you did)
- Refuse, then the no move to cold storage is done. 


I am not talking on the action itself, I am talking about the dialogue once you click the button / action.

Reply:
Ah, ok. So you want me to add a scenario where I cancel the move to coldstorage in the confirmation dialog. That wasn't what I understood at first.

Reply:
> Ah, ok. So you want me to add a scenario where I cancel the move to coldstorage in the confirmation dialog.
yep, thanks :) 

> That wasn't what I understood at first.
sorry about that. 



Thanks again for your efforts